internal-packages/next-dev/README.md

```diff
@@ -54,7 +54,11 @@ export default nextConfig;
 // (note: this check is recommended but completely optional)
 if (process.env.NODE_ENV === 'development') {
 	// we simply need to call the utility
-	setupDevPlatform();
+	// (`await`ing the call helps making sure that the setup has succeeded,
+	//  it is not mandatory but recommended. In config files using the
+	//  cjs format, where top level awaits are not supported you can
+	//  safely simply call the function and omit the `await`)
+	await setupDevPlatform();
```

Comment:
In files that don't support top-level await, an alternative to avoid the unhandled rejection could be:

```
setupDevPlatform().catch(e => console.error(e));
```

Reply:
yes, thanks for the input 🙂

do you think that it would be valuable to add in the comment? (or even in the code?)

I'm just afraid to get too verbose here....

what do you think?